apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/preview-template-title.js

```diff
@@ -11,7 +11,7 @@ const PreviewTemplateTitle = ( { title, transform } ) => (
 	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	<div className="editor-post-title" style={ { transform } }>
 		<div className="editor-post-title__block">
-			<textarea className="editor-post-title__input" value={ title } />
+			<textarea className="editor-post-title__input" value={ title } onChange={ () => {} } />
```

Comment:
could we use lodash `noop` here?

Reply:
Sure. Isn't that the same?

Reply:
I guess so, but we can delegate the maintenance of these functions to lodash. They will take over future changes, cross-browsing support, etc. mho.